wazuh/README.md

```diff
@@ -0,0 +1,156 @@
+# Agent Integration: wazuh
+
+## Overview
+
+[Wazuh][3] provides a comprehensive security solution that detects, analyzes, and responds to threats across multiple IT infrastructure layers. Wazuh collects telemetry from endpoints, network devices, cloud workloads, third-party APIs, and other sources for unified security monitoring and protection.
+
+This integration parses the following types of logs:
+- **vulnerability-detector** : Vulnerability events generated by Wazuh.
+- **malware-detector** : Rootcheck events generated by Wazuh for detecting any malware in the system.
+- **file-integrity-monitoring** : Events related to file changes like permission, content, ownership and attributes.
+- **docker** : Activity Events of docker container. 
+- **github** : Events from audit logs from github organizations.
+- **google-cloud** : Security events related to google cloud platform services.
+- **amazon** : Security events from amazon AWS services.
+- **office365** : Security events related to office365.
+- **system** : Events from services like FTPD, PAM, SSHD, syslog, Windows, dpkg, yum, sudo, su, wazuh and ossec along with internal events.
+
+Visualize detailed insights into these logs through the out-of-the-box dashboards.
+
+## Setup
+
+### Installation
+
+To install the Wazuh integration, run the following Agent installation command and the steps below. For more information, see the [Integration Management][4] documentation.
+
+**Note**: This step is not necessary for Agent version >= 7.58.0.
```

Comment:
```suggestion
**Note**: This step is not necessary for Agent version >= 7.61.0.
```

I think is a better estimation. But we can circle back on this in a separate PR once this gets merged in as it's not that big of a change